internal/core/build.sh

```diff
@@ -82,6 +83,9 @@ while getopts "p:t:s:f:o:ulrcghzme" arg; do
   n)
     BUILD_DISK_ANN="OFF"
     ;;
+  x)
```

Comment:
If BUILD_DISK_ANN is no longer in use, then let's remove the associated logic.

Reply:
No, `BUILD_DISK_ANN ` will be reserved to transfer option to knowhere. But we remove all `BUILD_DISK_ANN` in segcore because `VectorDiskIndex.cpp` will  process more than `DISKANN` index.